spec/support/new_relic_test_service.rb

```diff
@@ -1,7 +1,7 @@
 require 'rester'
 
 class NewRelicTestService < Rester::Service
-  use Rester::Middleware::NewRelic
+  use Rester::Service::Middleware::NewRelic
```

Comment:
I think this can just be `use Middleware::NewRelic`.  I'm not sure why I had fully namespaced this before.  That's how I'm wanting users to include it, instead of having to write out that full thing.